src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -1025,9 +1037,9 @@ EclipseGrid::EclipseGrid(const Deck& deck, const int * actnum)
 
                 for (std::size_t j = 0; j <= this->getNY(); j++) {
                     /*
-                      The theta value is supposed to go counterclockwise, starting at 'twelve o clock'.
```

Comment:
I think we should mimic the cylindrical coordinate behaviour also in the spiderweb - or?

Reply:
That is actually a question I forgot to ask about. What is the orientation supposed to be for theta equals zero? I know the code for the cylindrical case has it oriented along the Y axis, but I was under the impression that along the X axis was eclipse compatible. That could be wrong though, and it is easily changed back We should just decided what it should be and make sure both spiderweb and cylindrical cases do the same.

Reply:
I don't know for sure - but I *think* that the current implementation with starting along the Y axis mimicks Eclipse.

Reply:
Ok, I will stick to that for now then.